location_obfuscation/src/anonymity.rs

```diff
@@ -0,0 +1,184 @@
+use geo::GeodesicArea;
+use geo::Scale;
+use geo_types::LineString;
+use geo_types::{Coord, Rect};
+use rand::prelude::*;
+use rstar::AABB;
+use rusty_roads::AnonymityConf;
+
+#[derive(Debug)]
+pub enum AnonymityError {
+    ConversionError,
+    TrajectoryNotInEnvelope,
+    PointTransform(String),
+}
```

Comment:
er dette en fuldstændig liste af mulige fejl? hvis nej

```suggestion
#[non_exhaustive]
pub enum AnonymityError {
    ConversionError,
    TrajectoryNotInEnvelope,
    PointTransform(String),
}
```